lib/ex_huobi/margin/websocket/market_ws.ex

```diff
@@ -8,12 +8,14 @@ defmodule ExHuobi.Margin.WebSocket.MarketWs do
 
       def start_link(args \\ %{}) do
         subscription = args[:subscribe] || ["market.btcusdt.mbp.150"]
+        bbo_subscription = args[:bbo_subscribe] || ["market.btcusdt.bbo"]
```

Comment:
@quantd2 I don't think we need to introduce this at all. `args[:subscribe]` can be a list of channel already. 

`args[:subscribe]` can pontentially a list of `["market.btcusdt.mbp.150", "market.btcusdt.bbo"] -> then simply loop through each channel to subscribe.

Reply:
the thing is for snapshot event, I need to send a subscribe request `{"req": topic}` which will cause error for best bid/ask topic

Reply:
we can pattern match the specific event, and send the correct operation payload to that one right?